pennylane/devices/experimental/default_qubit_2.py

```diff
@@ -110,7 +113,19 @@ def supports_derivatives(
         # backpropagation currently supported for all supported circuits
         # will later need to add logic if backprop requested with finite shots
         # do once device accepts finite shots
-        return execution_config.gradient_method == "backprop"
+        if execution_config.gradient_method == "backprop":
+            return True
+
+        if execution_config.gradient_method == "adjoint":
+            if circuit is None:
+                return False
```

Comment:
```suggestion
                return True
```

See `Device.supports_derivatives` documentation: 
```
If ``circuit`` is not provided, then the method should return whether or not device derivatives exist for **any** circuit.
```